lib/queries/orgsGetByTag.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+module.exports = function (active, tagId) {
+  var query = [
+    'SELECT',
+    'organisations.id AS id,',
+    'organisations.name AS name,',
+    'organisations.logo_url AS logo_url,',
+    'organisations.active AS active',
+    'FROM organisations'
+  ];
+
+  if (!active && !tagId) {
+    return query.concat([
+      'ORDER BY organisations.name ASC',
+      ';'
+    ]).join(' ');
+  } else if (active && !tagId) {
+    return query.concat([
+      'WHERE organisations.active = ' + active,
+      'ORDER BY organisations.name ASC',
+      ';']).join(' ');
+  } else if (!active && tagId) {
```

Comment:
By different functions, do you mean different files/queries? Or can I have then in the same file, and separate them out as functions?